impeller/entity/entity_pass.cc

```diff
@@ -840,6 +856,19 @@ bool EntityPass::OnRender(
   return true;
 }
 
+void EntityPass::IterateAllElements(
```

Comment:
Consider moving to separate patch, unused

Reply:
Removed.